automatic/nomacs.portable/nomacs.portable.nuspec

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>nomacs.portable</id>
+    <version>3.4.0.20161205</version>
```

Comment:
@majkinetor is the package fix notation required here?  I thought the plan was to try to push 3.4.0?